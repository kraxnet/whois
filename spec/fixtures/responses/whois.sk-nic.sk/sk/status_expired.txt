% For more information on Whois status codes, please visit https://icann.org/epp
%
% This WHOIS service is provided by SK-NIC and only contains
% information pertaining to Internet domain names registered by our
% our customers. By using this service you are agreeing (1) not to use any
% information presented here for any purpose other than determining
% ownership of domain names, (2) not to store or reproduce this data in
% any way without SK-NIC's approval, (3) not to use any high-volume, automated, electronic processes
% to obtain data from this service. Abuse of this service is monitored and
% actions in contravention of these terms will result in being permanently
% blacklisted.
%
% Access to the whois service is rate limited.

Domain:                       i-novinar.sk
Registrant:                   ZONE-0002
Admin Contact:                ZONE-0002
Tech Contact:                 ZONE-0002
Registrar:                    ZONE-0002
Created:                      2012-08-08
Updated:                      2017-09-03
Valid Until:                  2017-10-13
Nameserver:                   ns1.regzone.cz
Nameserver:                   ns1.regzone.info
Nameserver:                   ns1.regzone.de
EPP Status:                   pendingDelete, serverHold, redemptionPeriod

Registrar:                    ZONE-0002
Organization:                 ZONER s.r.o.
Organization ID:              35770929
Phone:                        +421.263815272
Email:                        sk-nic@zoner.sk
Street:                       Jaskov√Ω rad 5
City:                         Bratislava
Postal Code:                  831 01
Country Code:                 SK
Created:                      2017-09-01
Updated:                      2017-09-29
