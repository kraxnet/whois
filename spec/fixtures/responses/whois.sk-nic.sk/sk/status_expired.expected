#status
  %s == :expired

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s %TIME{2012-08-08}

#updated_on
  %s %CLASS{time}
  %s %TIME{2017-09-03}

#expires_on
  %s %CLASS{time}
  %s %TIME{2017-09-01}

#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
